databricks/koalas/series.py

```diff
@@ -2294,6 +2294,56 @@ def apply(self, func, args=(), **kwds):
         wrapped = ks.pandas_wraps(return_col=return_sig)(apply_each)
         return wrapped(self, *args, **kwds).rename(self.name)
 
+    # TODO: not all arguments are implemented comparing to Pandas' for now.
+    def aggregate(self, func: Union[str, List[str]]):
+        """Aggregate using one or more operations over the specified axis.
+
+        Parameters
+        ----------
+        func : str or a list of str
+            function name(s) as string apply to series.
+
+        Returns
+        -------
+        scalar, Series
+            The return can be:
+            - scalar : when Series.agg is called with single function
+            - Series : when Series.agg is called with several functions
+
+        Notes
+        -----
+        `agg` is an alias for `aggregate`. Use the alias.
+
+        See Also
+        --------
+        databricks.koalas.Series.apply
+        databricks.koalas.Series.transform
+
+        Examples
+        --------
+        >>> s = ks.Series([1, 2, 3, 4])
+        >>> s.agg('min')
+        1
+
+        >>> s.agg(['min', 'max'])
+        min    1
+        max    4
+        Name: 0, dtype: int64
+        """
+        if isinstance(func, list):
+            if all((isinstance(f, str) for f in func)):
+                rows = OrderedDict((f, eval("self.{}()".format(f), dict(self=self))) for f in func)
+                return Series(rows)
+            else:
+                raise ValueError("If the given function is a list, it "
+                                 "should only contains function names as strings.")
+        elif isinstance(func, str):
+            return eval("self.{}()".format(func))
```

Comment:
We should avoid `eval()` as far as possible. `getattr(self, func)()` instead?

Reply:
@ueshin Thanks for review ueshin :) I totally agree. fixed it !